lighthouse-core/test/audits/timing-budget-test.js

```diff
@@ -80,30 +82,60 @@ describe('Performance: Timing budget audit', () => {
       expect(result.details.items).toHaveLength(2);
     });
 
-    it('works for all supported timing metrics', async () => {
-      const metrics = [
-        'first-contentful-paint',
-        'first-cpu-idle',
-        'interactive',
-        'first-meaningful-paint',
-        'max-potential-fid',
-        'estimated-input-latency',
-        'total-blocking-time',
-        'speed-index',
-      ];
-      await Promise.all(metrics.map(async (metric) => {
+    describe('timings metrics', () => {
+      it('work for all supported timing metrics', async () => {
+        const metrics = [
```

Comment:
worth trying to get this list from somewhere programmatically? feels like we're bound to make a typo in one of these lists but then again we don't want a noop test situation where we're asserting that the list we generate *is the list we generate* soooo 🤷‍♂️

maybe the answer is we just need type checking for these tests at some point